pkg/controllers/manila/manila.go

```diff
@@ -123,6 +136,62 @@ func (c *ManilaController) sync(ctx context.Context, syncCtx factory.SyncContext
 	return c.setEnabledCondition(ctx)
 }
 
+func (c *ManilaController) getFsGroupPolicy(ctx context.Context) storagev1.FSGroupPolicy {
+
+	// Set the fsGroupPolicy param to the default value
+	fsGroupPolicy := "None"
```

Comment:
The [doc for `fsGroupPolicy`]( https://kubernetes-csi.github.io/docs/support-fsgroup.html#csi-driver-fsgroup-support) says that:
> If undefined, fsGroupPolicy will default to ReadWriteOnceWithFSType, keeping the previous behavior.

`fsGroupPolicy` was previously undefined, so setting it to `None` here will change the behavior, isn't it?

Reply:
Exactly, but as per [1]  and the conversation in [2] the purpose if to set to `None` for manila by default.  

[1] https://github.com/kubernetes/cloud-provider-openstack/pull/1894
[2] https://github.com/kubernetes/cloud-provider-openstack/issues/1893